app/src/main/AndroidManifest.xml

```diff
@@ -9,9 +9,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".ViewActivityHistory"></activity>
+        <activity android:name=".ui.ViewActivityHistory"></activity>
         <activity
-            android:name=".MainActivity"
+            android:name=".ui.MainActivity"

```

Comment:
created a new package for mainactivity to provide more structure and hierarchy to the directory